content/terraform/v1.10.x/docs/language/functions/index.mdx

```diff
@@ -22,8 +22,27 @@ For more details on syntax, see
 [_Function Calls_](/terraform/language/expressions/function-calls)
 in the Expressions section.
 
-The Terraform language does not support user-defined functions, and so only
-the functions built in to the language are available for use. The documentation includes a page for all of the available built-in functions.
+# Provider-defined Functions
+
+The Terraform language does not support _user_-defined functions.
+However [providers can implement functions](/terraform/plugin/framework/functions)
+which users can use.
```

Comment:
```suggestion
You cannot define your own functions in the Terraform configuration language, but you can develop your own providers that expose functions. Refer to [Provider-defined functions](/terraform/plugin/framework/functions) for information about defining functions in custom providers.
```
Switch to active voice. Is "exposing functions" the right way to characterize the action?